agent/ansible/README

```diff
@@ -22,5 +22,77 @@ file for guidance.
 
 * Usage
 #+begin_src shell
-ansible-playbook -i /path/to/inventory/pbench-agent.hosts pbench-agent-install.yml
+ansible-playbook -i /path/to/inventory/pbench-agent.hosts pbench_agent_install.yml
 #+end_src
+
+
+* Installing roles from Ansible Galaxy
+
+The roles that the above playbooks use are available from Ansible
+Galaxy, as part of the pbench.agent collection. You can install the
+collection with
+
+   ansible-galaxy collection install pbench.agent
+
+The roles will be installed under
+
+    $HOME/.ansible/collections/ansible_collections/pbench/agent
+
+and you will need to add this path to Ansible's roles path in order
+for the playbooks to find them:
+
+    ANSIBLE_ROLES_PATH=$HOME/.ansible/collections/ansible_collections/pbench/agent:$ANSIBLE_ROLES_PATH
+    export ANSIBLE_ROLES_PATH
+
+or, more permanently, you can add it to the /etc/ansible/ansible.cfg
+file.
+
+You will still a playbook to install the pbench agent - here's the
+standard pbench_agent_install.yml from the pbench repo:
+
+--8<---------------cut here---------------start------------->8---
+---
+- name: install pbench-agent
+  hosts: servers
+  become: yes
+  become_user: root
+
+  # The default value ('production') can be overriddent by cenv, a host-specific
+  # inventory variable.
+  vars:
+    pbench_configuration_environment: "{{ cenv | default('production') }}"
+
+  roles:
+    - pbench_repo_install
+    - pbench_agent_install
+    - pbench_agent_config
+--8<---------------cut here---------------end--------------->8---
+
+Once the dust settles a bit, we will probably move the playbooks to Galaxy
+as well.
+
+* Installing the roles *TO* Ansible Galaxy
+
+- Update the version in pbench/agent/galaxy.yml: for bug fixes,
+  increment the last of the three numbers; for new functionality that
+  leaves things backwards-compatible, increment the middle of the
+  three numbers; for incompatible updates, increment the first number.
+
+- `make test' and make sure that there are no blocking errors/warning.
+
+- `make publish' to publish the new version on Ansible Galaxy: you
+  need to have an account on Galaxy and your account needs to have
+  admin privileges to the `pbench' namespace.
+
+Prereqs:
+
+- an account on galaxy.ansible.com - I use my Github account.
```

Comment:
s/I/typically, one can use a GitHub account for a Galaxy account./

Reply:
Will fix.